classes/phing/tasks/ext/NotifySendTask.php

```diff
@@ -74,13 +74,13 @@ public function getIcon()
     /**
      * Set to a true value to not execute notifysend command.
      *
-     * @param bool $silent Don't execute notifysend? True not to.
+     * @param string $silent Don't execute notifysend? Truthy value.
      *
      * @return void
      */
     public function setSilent($silent)
     {
-        $this->silent = (bool) $silent;
+        $this->silent = StringHelper::booleanValue($silent);
```

Comment:
It is not necessary to use `StringHelper::booleanValue()` here as it happens in the
[`IntrospectionHelper`](https://github.com/phingofficial/phing/blob/ac3cc900a34fd0a2647850f367268687135b030b/classes/phing/IntrospectionHelper.php#L326) as well. Just dropping the bool cast would be enough.